js/src/components/customer-effort-score-prompt/index.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import CustomerEffortScore from '@woocommerce/customer-effort-score';
+import { recordEvent } from '@woocommerce/tracks';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { LOCAL_STORAGE_KEYS } from '.~/constants';
+import localStorage from '.~/utils/localStorage';
+
+/**
+ * A CustomerEffortScore wrapper that uses tracks to track the selected
+ * customer effort score.
+ *
+ * @param {Object} props React component props.
+ * @param {string} props.eventContext Context to be used in the CES wrapper events.
+ * @param {string} props.label Text to be displayed in the CES notice and modal.
+ */
```

Comment:
💅 

Documentation about records is missing

See `js/src/components/contact-information/store-address-card.js`  (as an example)

We are using JSDOC documentation in a way that running `npm run doc:tracking` generates tracking docs automatically.

Reply:
Thanks, added in 2932545 manually as in GLA repo it seems we haven't had `npm run doc:tracking` to generate tracking docs.

Reply:
Thanks for pointing out that we don't have that yet here. Didn't see this https://github.com/woocommerce/google-listings-and-ads/issues/117 issue